modules/RTC/TraceablePeerConnection.js

```diff
@@ -945,6 +967,11 @@ TraceablePeerConnection.prototype.createAnswer
                 _fixAnswerRFC4145Setup(remoteDescription, localDescription);
                 answer.sdp = localDescription.raw;
 
+                if (answer && this.options._sdpAnswerDirection) {
```

Comment:
you don't need to check for `answer` it will be there or the live above would have thrown